docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/kubernetes.md

```diff
@@ -1,8 +1,81 @@
 ---
-title: "Kubernetes agents"
+title: "Running Dagster+ agents on Kubernetes"
 displayed_sidebar: "dagsterPlus"
 sidebar_position: 30
 sidebar_label: "Kubernetes"
 ---
 
-# Kubernetes agents
\ No newline at end of file
+# Running Dagster+ agents on Kubernetes
+
+This page provides instructions for running the [Dagster+ agent](dagster-plus/getting-started/whats-dagster-plus#Agents) on a [Kubernetes](https://kubernetes.io) cluster.
+
+## Installation
+
+
+### Prerequisites
+
+You'll need [a Kubernetes cluster. This can be a self-hosted Kubernetes cluster or a managed offering like [Amazon EKS](https://aws.amazon.com/eks/), [Azure AKS](https://azure.microsoft.com/en-us/products/kubernetes-service), or [Google GKE](https://cloud.google.com/kubernetes-engine).
```

Comment:
I would also include the Image registry as a prerequisite.

Language from the OG docs:

> Access to a container registry to which you can push images and from which pods in the Kubernetes cluster can pull images. This can be:
> 
> A self-hosted registry,
> A public registry such as [DockerHub](https://hub.docker.com/), or
> A managed offering such as [Amazon ECR](https://aws.amazon.com/ecr/), [Azure ACR](https://azure.microsoft.com/en-us/services/container-registry/#overview), or [Google GCR](https://cloud.google.com/container-registry)